templates/pages/service.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "Services",
+  "layout": [
+    {
+      "type": "widgets",
+      "content": [
+        {
+          "name": "Deployment Button",
+          "x": 0,
+          "y": 0,
+          "width": 3,
+          "height": 3,
+          "definition": "deploymentButton"
+        },
+        {
+          "name": "Deployments view",
+          "width": 12,
+          "height": 28,
+          "definition": "deploymentsView",
+          "configuration": {
+            "customPollingTime": 10,
+            "mapOpenByDefault": false,
+            "mapHeight": 300,
+            "fieldsToShow": [
+              "status",
+              "name",
+              "blueprintName",
+              "location",
+              "subenvironmentsCount",
+              "subservicesCount"
+            ],
+            "pageSize": 50,
+            "sortColumn": "created_at",
+            "sortAscending": null,
+            "filterId": "csys-service-filter",
+            "filterByParentDeployment": false
+          },
```

I suggest to keep just:
```suggestion
          "configuration": {
            "filterId": "csys-service-filter"
          },
```
as all other configuration parameters are the same as default values.